packages/edit-site/src/components/layout/style.scss

```diff
@@ -244,17 +254,3 @@
 	}
 }
 
-.edit-site-layout.has-fixed-toolbar {
-	// making the header be lower than the content
-	// so the fixed toolbar can be positioned on top of it
-	// but only on desktop
-	@include break-medium() {
-		.edit-site-site-hub {
-			z-index: 4;
```

Comment:
I was surprised to learn this is not needed. Still not 100% sure it's not something weird with my env or FF. Needs more testing.

Reply:
This was because of #51840